ee/modules/030-cloud-provider-openstack/docs/FAQ_RU.md

```diff
@@ -219,22 +249,6 @@ openstack flavor show m1.medium-50g -c disk
 +-------+-------+
 ```
 
-Если объем диска — нулевой, то необходимо использовать параметр [rootDiskSize](cr.html#openstackinstanceclass-v1-spec-rootdisksize).
-
-Если объем диска — ненулевой, то возможны разные ситуации.
-OpenStack-провайдеры могут использовать разные типы дисков, для root-диска. Это может быть как сетевой диск, так и локальный диск. Требуется уточнить тип диска в документации или в технической поддержке провайдера.
-
-Локальные диски имеют свои ограничения, одно из которых то, что ВМ не может мигрировать между гипервизорами. Однако, локальные диски, как правило, дешевле и быстрее сетевых дисков. Поэтому мы рекомендуем следующее:
-* Для master-узла предпочтительнее использовать сетевой диск.
-* Для ephemeral-узла можно использовать локальный диск.
-* Избегайте использования flavor'ов с указанным объемом диска и одновременным использованием параметра `rootDiskSize`. В этом случае cloud-провайдер может взимать плату за использование диска, указанного в flavor, и за дополнительный диск с размером заданным в `rootDiskSize`.
-
-С учетом изложенного выше:
-- Если `rootDiskSize` не установлен, будет использован ephemeral-диск размера, соответствующего указанному во flavor, и типа, который определён на стороне cloud-provider'а.
-- Если `rootDiskSize` указан, будет использован Cinder volume OpenStack (указанного размера с типом диска по умолчанию, который определён у cloud-provider'a). Но, в этом случае есть несколько нюансов:
-  * Для master-узла всегда будет использоваться тип диска, указанный в параметре [volumeTypeMap](cluster_configuration.html#parameters-masternodegroup-volumetypemap) OpenStackClusterConfiguration;
-  * При необходимости, можно [переопределить](#как-переопределить-тип-диска-по-умолчанию-cloud-провайдера) тип диска по умолчанию cloud-provider'a.
-
 ## Как переопределить тип диска по умолчанию cloud-провайдера?
 
 Если у cloud-провайдера доступно несколько типов, то вы можете указать тип диска по умолчанию для образа. Для этого необходимо указать название типа диска в метаданных образа. Тогда все ВМ, создаваемые из этого образа, будут использовать указанный диск.
```

Comment:
```suggestion
Если у cloud-провайдера доступно несколько типов, то вы можете указать тип диска по умолчанию для образа. Для этого необходимо указать название типа диска в метаданных образа. Тогда все ВМ, создаваемые из этого образа, будут использовать указанный тип диска.
```

@konstantin-axenov is it about network disk, or local disk, or both?

Reply:
This is about network disk only.